src/OmniSharp.MSBuild/SolutionSdkFileUtil.cs

```diff
@@ -0,0 +1,34 @@
+using MSB = Microsoft.Build;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+
+namespace OmniSharp.MSBuild
+{
+    class SolutionSdkFileUtil
+    {
+        static MSB.Evaluation.Project GetEvalutatedProject(string filePath
+            , IDictionary<string, string> globalProperties = null)
+        {
+            var projopt = new MSB.Definition.ProjectOptions();
+            var projectCollection = globalProperties != null ? new MSB.Evaluation.ProjectCollection(globalProperties)
+                :
+                new MSB.Evaluation.ProjectCollection();
```

Comment:
formatting: this conditional expression is strangely formatted. Could you change it to something like this?

```C#
var projectCollection = globalProperties != null
    ? new MSB.Evaluation.ProjectCollection(globalProperties)
    : new MSB.Evaluation.ProjectCollection();
```